fastats/scaling/scaling.py

```diff
@@ -99,6 +99,22 @@ def rank(A):
     return res
 
 
+def demean(A):
+    """
+    Subtract the mean from the supplied data column-wise.
+    """
+    assert A.ndim > 1
+
+    n = A.shape[1]
+    res = empty_like(A, dtype=np_float64)
+
+    for i in range(n):
```

Comment:
Does this work with `prange`?

Reply:
(as in, we could have a `_parallel` variant)

Reply:
Yep, all of the scaling functions behave well with prange (using 0.36 release candidate).  I will add parallel versions as suggested, good idea.